habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -720,6 +721,7 @@ def _create_recep_info(
         self, scene_id: str, ignore_handles: List[str]
     ) -> Dict[str, Receptacle]:
         if scene_id not in self._receptacles_cache:
+            # TODO: consume the filter file to limit loaded receptacle to the "active" set?
```

Comment:
Doing this in a separate PR in case it breaks other parts of the codebase. Will link here when ready.

Reply:
PR: https://github.com/facebookresearch/habitat-lab/pull/2053